Polyhedron/demo/Polyhedron/Scene.cpp

```diff
@@ -737,66 +742,64 @@ bool Scene::dropMimeData(const QMimeData * /*data*/,
                          int /*column*/,
                          const QModelIndex &parent)
 {
-    //gets the moving items
-    QList<Scene_item*> items;
-    QList<int> groups_children;
+  //gets the moving items
+  QList<Scene_item*> items;
+  QList<int> groups_children;
```

Comment:
these 3 lines are just changed by indentation, please put them back to their initial position